lib/intrinsicclass.ts

```diff
@@ -96,13 +96,13 @@ export function MakeIntrinsicClass(
     });
   }
   for (const prop of Object.getOwnPropertyNames(Class)) {
-    const desc = Object.getOwnPropertyDescriptor(Class, prop);
+    const desc = Object.getOwnPropertyDescriptor(Class, prop) as PropertyDescriptor;
```

Comment:
Out of curiosity, why doesn't `Object.getOwnPropertyDescriptor` have type `PropertyDescriptor` already?

Reply:
This is probably somewhere that I'd use `!` instead of `as`, but it sounds like there's an ESLint rule against that?

Reply:
Yeah. I ended up checking in a change here that's the same as the other case you commented about: turn off the ESLint rule for this line (and a few lines below in a similar loop) and add a comment explaining why.